hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/EntitiesConfig.java

```diff
@@ -24,4 +24,5 @@
 public record EntitiesConfig(
         @ConfigProperty(defaultValue = "3153600000") @NetworkProperty long maxLifetime,
         // @ConfigProperty(defaultValue = "FILE") Set<EntityType> systemDeletable
-        @ConfigProperty(defaultValue = "false") @NetworkProperty boolean limitTokenAssociations) {}
+        @ConfigProperty(defaultValue = "false") @NetworkProperty boolean limitTokenAssociations,
+        @ConfigProperty(defaultValue = "true") @NetworkProperty boolean unlimitedAutoAssociations) {}
```

Comment:
Should this be `unlimitedAutoAssociationsEnabled` or `allowUnlimitedAutoAssociations` for clarity

Reply:
I will change it to `allowUnlimitedAutoAssociations`

Reply:
I think it should be `unlimitedAutoAssociationsEnabled`, because we use this pattern more frequently.